lane.py

```diff
@@ -105,6 +105,12 @@ class BaseFacets(FacetConstants):
     This is intended solely for use as a base class.
     """
 
+    # If the use of a certain faceting object has implications for the
+    # type of feed (the way FeaturedFacets always implies a 'groups' feed),
+    # set the type of feed here. This will override any CACHED_FEED_TYPE
+    # associated with the WorkList.
+    CACHED_FEED_TYPE = None
+
```

Comment:
Making sure that CACHED_FEED_TYPE is always present for WorkLists and Facets (even if the value is None) lets me take `cache_type` out of the `CachedFeed.fetch` method signature. We can always figure out the feed type by looking at the WorkList and the Facets.